packages/create-subscription/src/__tests__/createSubscription-test.internal.js

```diff
@@ -0,0 +1,423 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @emails react-core
+ */
+
+'use strict';
+
+let createSubscription;
+let ReactFeatureFlags;
+let React;
+let ReactNoop;
+
+describe('createSubscription', () => {
+  beforeEach(() => {
+    jest.resetModules();
+    createSubscription = require('create-subscription').createSubscription;
+    ReactFeatureFlags = require('shared/ReactFeatureFlags');
+    ReactFeatureFlags.debugRenderPhaseSideEffectsForStrictMode = false;
+    React = require('react');
+    ReactNoop = require('react-noop-renderer');
+  });
+
+  // Mimics a partial interface of RxJS `BehaviorSubject`
```

Comment:
Nit: Why not pull in the real RxJS as a dev dependency?

Reply:
Yeah, I guess I could. I considered doing this at one point 😄